src/py21cmfast/wrapper.py

```diff
@@ -765,8 +769,8 @@ def _get_photon_nonconservation_data():
         "Q_analytic",
         "z_calibration",
         "nf_calibration",
-        "delta_z_photon_cons",
         "nf_photoncons",
+        "delta_z_photon_consnf_photoncons",
```

Comment:
This looks like it needs a comma!!

Reply:
Not a comma, but delete some text. Nice catch, trying to do things too quickly...